### ISSUE TITLE
Add to phpstan org

### ISSUE THREAD

**Issue by Fuco1** [Fuco1/phpstan-kdyby-doctrine#2]
It would be nice to have this in the phpstan org and also placed on packagist so it is easier to include in projects.

@ondrejmirtes Any issues I should fix before we can move it?

**Comment by ondrejmirtes** [comment]
Hi, thank you for making this extension!

I'd like if there was some build system (like Phing or Make) in place so one could run all the checks locally with a single command. The `script` section in .travis.yml would look like this:

```
script:
  - vendor/bin/phing
```

That can be achieved by adopting `build.xml` file in the codebase and `phing/phing` in the depedencies.  You can get inspired for example here: https://github.com/phpstan/phpstan-nette/blob/master/build.xml

I also suspect that currently you're not running PHPUnit tests at all - I see lint, phpcs and phpstan, but not phpunit in your .travis.yml.

When you add build, I have no problem making this extension official :) Looking forward to that!

**Comment by Fuco1** [comment]
Well the tests are performed by running phpstan, sort of: if the code stops working the analysis of `tests` directory will produce errors.

I'll add phing because it's better than make but I still don't quite like it. XML :/

**Comment by ondrejmirtes** [comment]
But you're not running the tests! :)

**Comment by Fuco1** [comment]
Those tests are not phpunit tests but regular php files analyzed with phpstan.  If there is a bug in phpstan then it will report errors in those files:

    - ./vendor/bin/phpstan analyze -l 7 --configuration=tests/phpstan.neon src tests # < tests

If everything passes it means that the extension is working.  So the testing is indirect, not unit testing but more like integration testing.

**Comment by ondrejmirtes** [comment]
Oh, I see :) That's clever!

**Comment by Fuco1** [comment]
This is probably so out-dated that it is not worth continuing.  Kdyby Doctrine is semi-abandoned.  I'm closing this as I don't have enough time to finish and the impact is minimal.
